datasets/fec/pipelines/_images/run_csv_transform_kub/csv_transform.py

```diff
@@ -80,19 +91,19 @@ def main(
             df["transaction_dt"] = df["transaction_dt"].astype(str)
             date_for_length(df, "transaction_dt")
             df = resolve_date_format(df, "transaction_dt", pipeline_name)
-
         elif "other_committee_tx_20" in pipeline_name:
+            df.columns = csv_headers
             df["transaction_dt"] = df["transaction_dt"].astype(str)
             date_for_length(df, "transaction_dt")
             df = resolve_date_format(df, "transaction_dt", pipeline_name)
-
         elif "opex" in pipeline_name:
             df = df.drop(columns=df.columns[-1], axis=1)
+            df.columns = csv_headers
             df["transaction_dt"] = df["transaction_dt"].astype(str)
             date_for_length(df, "transaction_dt")
             df = resolve_date_format(df, "transaction_dt", pipeline_name)
-
         else:
+            df.columns = csv_headers
             pass
 
         logging.info(f"Saving to output file.. {target_file}")
```

Comment:
Move logging to respective function definition.

Reply:
done